### PR TITLE
Documentation in Upcoming event and Minor changes in Home Pages

### PR DESCRIPTION
**Pull request by shreyakapoor08**
Fix #554 

Documentation in Upcoming event and Minor changes in Home Pages

image

image

### CODE REVIEW

**Comment by shreyakapoor08** [comment]
@smaranjitghose Please check :)

File: docs/_build/html/_sources/announcement.rst.txt [hunk 1]
```diff
@@ -1,2 +1,31 @@
...
+.. image:: ./images/announcementpage/upcomingevent.JPG
+  :width: 600
+  :align: center
+  :alt: Alternative text
+
+If you want to make changes in this section, carry out the following changes in code:
+
+- To make changes in the heading - Go to `announcement.html <https://github.com/smaranjitghose/girlscript_chennai_website/blob/master/announcement.html>`__ and search :python:`recent`, you will see <h2> tag where you can make changes.
+
+* To make changes in the styling of heading - Go to `announcement.css <https://github.com/smaranjitghose/girlscript_chennai_website/blob/master/css/announcement.css>`__ and search :python:`.recent` and do the desired changes.
```
Comment (smaranjitghose) [inline docs/_build/html/_sources/announcement.rst.txt:21]: We don't want the user to bother about changing the style but rather the content. Hence this is not required. (As we have already taken care of that). Make similar reductions in the home page wherever necessary

File: docs/_build/html/_sources/announcement.rst.txt [hunk 2]
```diff
@@ -1,2 +1,31 @@
...
+
+- To make changes in the content of cards - Go to `announcement.html <https://github.com/smaranjitghose/girlscript_chennai_website/blob/master/announcement.html>`__ and search :python:`timeline-content`, you will see <h2> and <p> tags where you can make changes in the heading and the description respectively.
+
+* To make changes in the styling of cards - Go to `announcement.css <https://github.com/smaranjitghose/girlscript_chennai_website/blob/master/css/announcement.css>`__ and search :python:`.timeline-content` and do the required changes.
```
Comment (smaranjitghose) [inline docs/_build/html/_sources/announcement.rst.txt:25]: We don't want the user to bother about changing the style but rather the content. Hence this is not required. (As we have already taken care of that). Make similar reductions in the home page wherever necessary

File: docs/_build/html/_sources/announcement.rst.txt [hunk 3]
```diff
@@ -1,2 +1,31 @@
...
+
+- To make changes in the button - Go to `announcement.html <https://github.com/smaranjitghose/girlscript_chennai_website/blob/master/announcement.html>`__ and search :python:`card-stats`, you will see <a> tag where you can change the path in the href.
+
+* To add an additional event - Go to `announcement.html <https://github.com/smaranjitghose/girlscript_chennai_website/blob/master/announcement.html>`__ and search :python:`timeline-container right` where you can copy the whole <div> and just replace the :python:`right` class with :python:`left` so that the added card is shown on the left hand side, then you can change it's heading and description of the event under <h3> and <p> pag respectively.
```
Comment (smaranjitghose) [inline docs/_build/html/_sources/announcement.rst.txt:29]: Nice! 😎

**Comment by shreyakapoor08** [comment]
@smaranjitghose Done :)

**Pull request merged**